packages/@glimmer/global-context/index.ts

```diff
@@ -179,6 +189,8 @@ export default function setGlobalContext(context: GlobalContext) {
   warnIfStyleNotTrusted = context.warnIfStyleNotTrusted;
   assert = context.assert;
   deprecate = context.deprecate;
+
+  assign(FEATURES, context.FEATURES);
```

Comment:
I ran into the Object.assign error that was mentioned here as well: https://github.com/glimmerjs/glimmer-vm/pull/1338#issuecomment-924446617 

Do we still support IE11 here or can we just [remove the code that triggers the error](https://github.com/glimmerjs/glimmer-vm/blob/4f1bef0d9a8a3c3ebd934c5b6e09de4c5f6e4468/test/index.html#L42-L46)?

Reply:
We don't need to support IE11, but anyway with the change proposed above this is no longer relevant.